app/views/teams/_team_summary.html.haml

```diff
@@ -10,7 +10,7 @@
   %div.span5
     %h3.muted= t('teams.summary.solved_challenges_header')
     - if @team.solved_challenges.size == 0
-      .zero-items-text= t('teams.summary.zero_items_text')
+      .zero-items-text{:id => "noSolvedChallengesText"}= t('teams.summary.zero_items_text')
```

Comment:
It shouldn't be necessary to add an ID to this just for the tests, you should be able to just grab it by the class.